app/controllers/api/audit_logs_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::AuditLogsController < Api::ApplicationController
+  skip_after_action :verify_policy_scoped, only: :index
+
+  def index
+    audit_logs = Audited::Audit.all.order(created_at: :desc)
```

Comment:
If the application generates a large number of audit trails, loading all of them at once could lead to performance problems (slow response times, high memory usage on the server). This is a common concern with audit log tables, as they can grow very large over time.

Reply:
Pagination likely required, if there is even a need to serve this index endpoint.